client/lib/hoverIntent/README.md

```diff
@@ -0,0 +1,64 @@
+## Hoverintent
+
+"hoverIntent is a plug-in that attempts to determine the user's intent"
```

Comment:
This could use a better description. I wish we really had a plugin to determine user's intent for real! 😆 

Also, it's no longer a plugin if it's in `/lib`, right?